### PR TITLE
- Update to ungoogled-chromium 103.0.5060.134-1

### PR DESCRIPTION
**Pull request by teeminus**
No upates of the patches needed.

Do we want to set the `is_official_build` flag with this release like requested in #163?

### CODE REVIEW

**Comment by teeminus** [comment]
I do. And when updating the patches I always have the flag set. There is a dedicated branch in my fork with the flag set, so enabling the flag with this PR is just a matter of changing the source branch.

**Comment by PF4Public** [comment]
@Nifury Have you also tried `is_official_build` on Windows?

**Comment by teeminus** [comment]
From my experience, building and liking takes a bit more time and resources. The question is if it will work in the CI or if we run in any limitations there.

**Comment by PF4Public** [comment]
> From my experience, building and liking takes a bit more time and resources. The question is if it will work in the CI or if we run in any limitations there.

Exactly!

**Review by Nifury — APPROVED** [review]

**Comment by networkException** [comment]
Have any of you made a rebuild of this? There was a issue with a third party binary (see https://github.com/ungoogled-software/ungoogled-chromium/issues/2019)

**Comment by teeminus** [comment]
> Have any of you made a rebuild of this? There was a issue with a third party binary (see https://github.com/ungoogled-software/ungoogled-chromium/issues/2019)

Can you rephrase that, I don't understand your question.

**Comment by networkException** [comment]
So someone else publishing ungoogled-chromium binaries for windows apparently had some issues with the latest build. I want to make sure that we don't run into the same issue.

As such I asked if any of you have made a working build of this pull request yet

**Comment by teeminus** [comment]
image

Of course, I always compile before creating a PR to make sure it works.

**Comment by networkException** [comment]
Alright perfect, thank you :)

**Review by networkException — APPROVED** [review]

**Comment by PF4Public** [comment]
So, was there a definitive decision on `is_official_build` flag for Windows?

**Comment by teeminus** [comment]
@Nifury has a CI job with the flag set running, I think we are waiting for that to finish.

**Comment by Nifury** [comment]
@teeminus Build failed. https://github.com/Nifury/ungoogled-chromium-windows/runs/7463225940?check_suite_focus=true#step:5:4959

I think it's because `windows-fix-generate-resource-allowed-list.patch` passes bytes to subprocess.Popen. 
```python
names.add(sym_name.encode('utf8'))
...
for name in names:
    undname = subprocess.Popen(['undname', name],
```
It's not an issue in Python 3.8 or later (`Changed in version 3.8: args parameter accepts a path-like object if shell is False and a sequence containing bytes and path-like objects on Windows.`), but the CI job uses Python 3.7.

**Comment by teeminus** [comment]
Yeah, I already saw that. I think we can fix that with the next release, so we can merge this PR for now.

**Comment by Nifury** [comment]
Sure!

**Comment by Nifury** [comment]
https://github.com/Nifury/ungoogled-chromium-windows/actions/runs/2724307448
Builds and runs fine. Just need to remove the `encode('utf-8')` part.

**Comment by teeminus** [comment]
Shall we create a -2 or do we wait for the next release?

**Comment by Nifury** [comment]
I'm fine with either, but why not create a -2. It gives a clearer commit history.

**Comment by teeminus** [comment]
Do you want to create a PR for your fixed branch or shall I fix mine and create a PR afterwards?

**Comment by Nifury** [comment]
It's probably better that you create a PR. You added a patch for `is_official_build` after all.

**Comment by teeminus** [comment]
Okay, then I will create a PR tomorrow with a cleaned up branch. Otherwise there will be a lot of merge commits from that branch being added to master 😆

**Comment by teeminus** [comment]
I was thinking about if we really want to bump the revision with the PR. It should not change the user experience so I thought we don't need a new release for that.

**Comment by Nifury** [comment]
Maybe the official build runs negligibly faster than the dev build...
Sure we can wait for the next release if you prefer.

**Comment by teeminus** [comment]
What do @PF4Public and @networkException think, should we bump the revision and create a new release with the release flag set or wait for the next core release?

**Comment by networkException** [comment]
I'd be in favor of making a release now.

If we were to wait for the next release it might not be clear that an issue was actually caused by `is_official_build` and not changes in upstream chromium or the patchset.
